css/index.css

```diff
@@ -94,7 +98,7 @@ header nav a:hover {
 
 header .logo {
     align-self: flex-end;
-    margin: 28px 0 0 91px;
+    margin: 28px 0 0 11.3%;
```

Comment:
Great use of percentages in the margin, and understanding how inheritance effects the particular percentage of each element being scaled.